sdk/storage/azure-storage-queue/azure/storage/queue/_generated/_serialization.py

```diff
@@ -100,14 +100,6 @@ def deserialize_from_text(cls, data, content_type=None):
                 raise DeserializationError("JSON is invalid: {}".format(err), err)
         elif "xml" in (content_type or []):
             try:
-
```

Comment:
Anything in the `_generated` folder is automatically generated and so we cannot modify it manually. So unfortunately, we'll have to leave these until the code generator fixes them.

Can you please revert anything in this folder?